### PR TITLE
Migrate all to pagy

### PR DESCRIPTION
**Pull request by innatewonderer**

### CODE REVIEW

**Comment by innatewonderer** [comment]
Spec that's failing on this PR has been fixed in e2424bcd010821f2c49cd88998c03db9680d0097

File: app/views/playlists/index.html.erb [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
 <% if authorized? %>
-<div class="additional_options_solo">
-  <div class="button_wrapper">
-  <%= link_to image_tag("icons/plus.png") + "create new playlist", new_user_playlist_path(@user), id: 'create_new_playlist', class: 'button' %>
-  </div>
-  <% if @user.playlists.only_public.size > 1 %>
+  <div class="additional_options_solo">
     <div class="button_wrapper">
-    <%= link_to image_tag("icons/arrows.png") + "change playlist order", sort_user_playlists_path(@user.login), id: 'change_playlist_order', class: 'button'  %>
+      <%= link_to image_tag("icons/plus.png") + "create new playlist", new_user_playlist_path(@user), id: 'create_new_playlist', class: 'button' %>
```
Comment (innatewonderer) [inline app/views/playlists/index.html.erb]: The change is displayed incorrectly... 
It's showing as if I renamed things, but really it's just spacing cleanup.

**Comment by sudara** [comment]
@innatewonderer Sweet! Looks like specs are still failing, some lines in an initializer that should be removed:

```
NameError: uninitialized constant WillPaginate
/home/travis/build/sudara/alonetone/config/initializers/misc.rb:6:in `<top (required)>'
```

**Comment by innatewonderer** [comment]
`Radio` apparently needs more love ... a.k.a giving me problems :) So changing status to `not ready`

File: app/views/assets/radio.html.erb [hunk 2]
```diff
@@ -1,34 +1,34 @@
...
   </div>
-    <div id="radio_tracks" class="box">
-        <%= render partial: 'assets/asset', collection: @assets  %>
-    </div>
-    <div class="mini_paginator">
-      <%= will_paginate @assets %>
-    </div>
 <% end %>
 <% content_for :right do %>
-    <div id="radio" class="box">
```
Comment (innatewonderer) [inline app/views/assets/radio.html.erb]: Just styling here.

File: app/views/playlists/_your_favorites.html.erb [hunk 3]
```diff
@@ -1,6 +1,6 @@
 <div id="your_favorites" class="playlist_source">
   <div class="mini_paginator">
-    <%= will_paginate @favorites, {param_name: :favorites_page, ignore_get_params: true}%>
```
[the remark this answers is not in the document]
Comment (innatewonderer) [reply app/views/playlists/_your_favorites.html.erb]: @sudara Just noticing that I'm actually deleting it in a few places... and I don't see any reference in will_paginate. LMK if you know if I should worry about it, or OK to just delete.

[the remark this answers is not in the document]
Comment (sudara) [reply app/views/playlists/_your_favorites.html.erb]: I actually have no idea what `ignore_get_params` is! Maybe it was an old option of will_paginate. I wouldn't worry about it!

**Comment by sudara** [comment]
@innatewonderer Looking good! Super exciting to get rid of that gem :D 

I'm ready to merge, except for the fact that the last commit looks like it belongs on your other (paranoia) PR... hence the failing spec...

**Comment by sudara** [comment]
Goodbye will paginate! It was nice knowing you...

**Pull request merged**